Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Source.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob.Definitions
+{
+    public class Source
+    {
+        public AuthenticationMethod AuthenticationMethod { get; set; }
+
+        /// <summary>
+        ///     The base URI for the storage account.
+        ///     Use either URI and SAS Token or Connection string.
```

Comment:
I think this line can be removed, since the user has already selected the authentication method.